pkg/metrics/controller_metrics.go

```diff
@@ -110,6 +114,20 @@ var (
 			Aggregation: view.LastValue(),
 			TagKeys:     []tag.Key{keyName, keyFleetName, keyNamespace},
 		},
+		{
+			Name:        fleetCountersName,
+			Measure:     fleetCountersStats,
+			Description: "Aggregated Counters counts and capacity across GameServers in the Fleet",
+			Aggregation: view.LastValue(),
+			TagKeys:     []tag.Key{keyFleetName, keyNamespace, keyType, keyCounter},
```

Comment:
```suggestion
			TagKeys:     []tag.Key{keyName, keyNamespace, keyType, keyCounter},
```

Since `fleet/counters` matches to `fleets/replicas_count` i think we should match on name key as well.

Reply:
Yeah, I'm not sure as to why `fleets/replicas_count` uses `keyName` for recording the fleetname https://github.com/googleforgames/agones/blob/0dc9654a173c7e77a80bdf85e4a8561d105b67c4/pkg/metrics/controller.go#L306

Where fleetautoscaler uses `keyFleetName` for recording the fleetname https://github.com/googleforgames/agones/blob/0dc9654a173c7e77a80bdf85e4a8561d105b67c4/pkg/metrics/controller.go#L175-L176

Reply:
I figure the idea was because `keyName` is the actual name of the backing resources. So on `fleets/replica_count` the name of the resource _is_ the fleet name, so using `keyFleetName` is kinda redundant.

At this point, i figure we should be consistent with the metrics that already exist.

Reply:
Sounds good, I'll change it to keyName.